### PR TITLE
Updated EWF  http_network, websocket, httpd to use the latest EiffelN…

### PR DESCRIPTION
**Pull request by jvelilla**
…et SSL

version.

### CODE REVIEW

File: library/network/http_network/src/ssl/http_stream_secure_socket.e [hunk 1]
```diff
@@ -55,12 +55,19 @@ feature -- Secure connection Helpers
...
-	set_secure_protocol_to_tls_1_0
+ 			-- Set `ssl_protocol' with `Ssl_23'.
+ 			-- Protocol not supported anymore.
+ 		obsolete
+			"Use set_secure_protocol_to_tls_1_2 [2017-11-30]."
+ 		local
+ 			err: DEVELOPER_EXCEPTION
+ 		do
+ 		    create err
+ 			err.set_description ("SSL_2 or SSL_3 are not supported anymore, upgrate to TLS set_secure_protocol_to_tls_1_2")
```
Comment (jocelyn) [inline library/network/http_network/src/ssl/http_stream_secure_socket.e]: bad indentation

File: library/network/http_network/src/ssl/http_stream_secure_socket.e [hunk 2]
```diff
@@ -176,7 +183,14 @@ feature -- Output
 		end
 
 note
-	copyright: "2011-2013, Javier Velilla, Jocelyn Fiat and others"
+	copyright: "2011-2017, Jocelyn Fiat, Javier Velilla, Olivier Ligot, Colin Adams, Eiffel Software and others"
```
Comment (jocelyn) [inline library/network/http_network/src/ssl/http_stream_secure_socket.e]: Maybe keep previous copyright

File: library/network/websocket/client/src/ssl/web_socket_client.e [hunk 3]
```diff
@@ -27,8 +27,8 @@ feature -- Factory
 				Result := l_secure
 				if attached secure_protocol as l_prot then
 					if l_prot.is_case_insensitive_equal ("ssl_2_3") then
-						l_secure.set_secure_protocol_to_ssl_2_or_3
-					elseif l_prot.is_case_insensitive_equal ("tls_1_0") then
+ 						l_secure.set_secure_protocol_to_ssl_2_or_3
```
Comment (jocelyn) [inline library/network/websocket/client/src/ssl/web_socket_client.e]: remove space at the beginning of the lines

File: library/server/httpd/configuration/httpd_configuration_i.e [hunk 4]
```diff
@@ -287,19 +287,19 @@ feature -- Element change
...
 		end
 
 	set_secure_protocol_from_string (a_ssl_version: READABLE_STRING_GENERAL)
-			-- Set `secure_protocol' with `a_ssl_version'
+			-- Set `secure_protocol' with `a_ssl_version'.
 		do
 			if a_ssl_version.is_case_insensitive_equal ("ssl_2_3") then
-				set_secure_protocol_to_ssl_2_or_3
-			elseif a_ssl_version.is_case_insensitive_equal ("tls_1_0") then
+ 				set_secure_protocol_to_ssl_2_or_3
```
Comment (jocelyn) [inline library/server/httpd/configuration/httpd_configuration_i.e]: remove space at the beginning of the lines

File: library/server/httpd/configuration/httpd_configuration_i.e [hunk 5]
```diff
@@ -316,6 +316,8 @@ feature -- SSL Helpers
 
 	set_secure_protocol_to_ssl_2_or_3
 			-- Set `secure_protocol' with `Ssl_23'.
+		obsolete
+				"Use set_secure_protocol_to_tls_1_0 [2017-11-30]."
```
Comment (jocelyn) [inline library/server/httpd/configuration/httpd_configuration_i.e]: Bad indentation,
Note sure we should use date 2017-11-30 or Current date.

File: library/server/httpd/ssl/httpd_configuration.e [hunk 6]
```diff
@@ -36,10 +36,17 @@ feature -- Access
...
-		end
+ 			-- Set `ssl_protocol' with `Ssl_23'.
+ 			-- Protocol not supported anymore.
+ 		obsolete
+			"Use set_secure_protocol_to_tls_1_2 [2017-11-30]."
+ 		local
+ 			err: DEVELOPER_EXCEPTION
+ 		do
+ 		    create err
+ 			err.set_description ("SSL_2 or SSL_3 are not supported anymore, upgrate to TLS set_secure_protocol_to_tls_1_2")
```
Comment (jocelyn) [inline library/server/httpd/ssl/httpd_configuration.e]: bad indentation

File: library/server/httpd/ssl/httpd_configuration.e [hunk 7]
```diff
@@ -36,10 +36,17 @@ feature -- Access
...
 
 	set_secure_protocol_to_ssl_2_or_3
-			-- Set `secure_protocol' with `Ssl_23'.
-		do
-			set_secure_protocol ({SSL_PROTOCOL}.Ssl_23)
-		end
+ 			-- Set `ssl_protocol' with `Ssl_23'.
+ 			-- Protocol not supported anymore.
+ 		obsolete
+			"Use set_secure_protocol_to_tls_1_2 [2017-11-30]."
```
Comment (jocelyn) [inline library/server/httpd/ssl/httpd_configuration.e]: maybe use current date for obsolete timestamp

File: library/network/http_network/src/ssl/http_stream_secure_socket.e [hunk 8]
```diff
@@ -55,12 +55,19 @@ feature -- Secure connection Helpers
...
-			-- Set `ssl_protocol' with `Ssl_23'.
-		do
-			set_secure_protocol ({SSL_PROTOCOL}.Ssl_23)
-		end
-
-	set_secure_protocol_to_tls_1_0
+ 			-- Set `ssl_protocol' with `Ssl_23'.
+ 			-- Protocol not supported anymore.
+ 		obsolete
+			"Use set_secure_protocol_to_tls_1_2 [2017-11-30]."
```
Comment (jocelyn) [inline library/network/http_network/src/ssl/http_stream_secure_socket.e]: maybe use current date for obsolete timestamp

File: library/network/http_network/src/ssl/http_stream_secure_socket.e (hunk 2 above)
Comment (jvelilla) [inline library/network/http_network/src/ssl/http_stream_secure_socket.e]: I don't have anymore the previous lic file.

**Pull request merged**